### PR TITLE
Cloud changelog Feb 24th release

### PR DESCRIPTION
**Pull request by amyblais**
Add known issues

### CODE REVIEW

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 1]
```diff
@@ -2,6 +2,48 @@
...
+
+#### Support Packet Generation
+ - Allows a System Admin to download a support packet which provides helpful information to our internal support team.
+
+### Improvements
+
+#### User Interface (UI)
+ - Removed the 5-page limit on previewing PDFs.
+ - Added "files" as a reserved team name.
+ - Searching for a channel url_name now shows the channel.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:20]:
```suggestion
 - Searching for a channel by URL now returns the channel.
```

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 2]
```diff
@@ -2,6 +2,48 @@
...
+ - Users are now provided with feedback when creating a custom category name for a channel that exceeds the character limit.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:21]:
```suggestion
 - Users are now provided with feedback when creating a custom category name that exceeds the character limit.
```

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 3]
```diff
@@ -2,6 +2,48 @@
...
+
+#### Administration
+ - Added support for compressed export files with attachments.
+ - Server crashes due to runtime panics are now captured as a log line.
+ - Optimized Direct Message creation by fetching all users involved in a single Database call.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:26]:
```suggestion
 - Optimized Direct Message creation by fetching all users involved in a single database call.
```

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 4]
```diff
@@ -2,6 +2,48 @@
...
+ - A change in a user's ``NotifyProps`` during the user import process will not send an email notification. This is done to make it consistent with other parts of the import process where a change in user's attributes would also not send any notifications.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:27]:
```suggestion
 - During the user import process, a change in a user's ``NotifyProps`` will not send an email notification. This is done to make it consistent with other parts of the import process where a change in user's attributes would also not send any notifications.
```

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 5]
```diff
@@ -2,6 +2,48 @@
...
+ - Implemented a job to delete unused export files.
+
+### Bug Fixes
+ - Fixed an issue where demoting a user to a guest would not take effect in an environment with read replicas.
+ - Fixed an issue where creation of a bot would fail due to replica lag.
+ - Fixed an issue where ``mmctl channel move`` did not allow moving private channels.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:33]:
```suggestion
 - Fixed an issue where ``mmctl channel move`` did not allow private channels to move.
```

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 6]
```diff
@@ -2,6 +2,48 @@
...
+ - Fixed an issue where markdown tables did not wrap correctly.
+ - Fixed an issue where the search bar styling on dark themes was off on mobile web view.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:35]: What does "off" refer to in this specific context? Does it indicate disabled or something else?

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 7]
```diff
@@ -2,6 +2,48 @@
...
+ - Fixed an issue where the **Main Menu** on webapp appeared more left-aligned than previously.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:36]:
```suggestion
 - Fixed an issue where the **Main Menu** on webapp appeared more left-aligned than previous releases.
```

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 8]
```diff
@@ -2,6 +2,48 @@
...
+ - Fixed an issue where sticky sidebar headings appeared under **More Unreads**.
+ - Fixed an issue where the group channel icon was misaligned in the channel switcher.
+ - Fixed an issue where line breaks were ignored when used with inline images.
+ - Fixed a panic when the Oauth discovery endpoint would not return a Cache-Control header.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:40]:
```suggestion
 - Fixed a panic when the OAuth discovery endpoint would not return a Cache-Control header.
```

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 9]
```diff
@@ -2,6 +2,48 @@
...
+
+### Known Issues
+ - Sometimes an "Unable to get role" error appears when changing a channel member role in **System Console > User Management > Channel**.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:43]:
```suggestion
 - Sometimes an "Unable to get role" error appears when changing a channel member role in **System Console > User Management > Channels**.
```

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 10]
```diff
@@ -38,8 +80,6 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 ### Known Issues
  - The slash command autocomplete options cover the input box on some reply threads on the right-hand side.
- - Setting changes do not get saved in **System Console > Site Configuration > Public Links**.
- - Channel header text "This channel has guests" is misaligned.
  - Sometimes an "Unable to get role" error appears when changing a channel member role in **System Console > User Management > Channel**.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:83]:
```suggestion
 - Sometimes an "Unable to get role" error appears when changing a channel member role in **System Console > User Management > Channels**.
```

**Review by cwarnermm — COMMENTED** [review]
Suggestions provided inline.

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 11]
```diff
@@ -2,6 +2,48 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-02-24
+
+### Highlights
+
+#### Custom Statuses
+ - Users can set a custom status by using the custom status dropdown in the status menu.
```
Comment (esethna) [inline source/cloud/cloud-administration/cloud-changelog.md:10]: Custom Statuses allows users to add a descriptive status message and emoji that’s visible to everyone. Users now gain the flexibility to express their current status in any way they prefer. Mobile support is coming soon.

**Review by esethna — APPROVED** [review]
looks good, just minor wording tweak

**Review by wiersgallak — APPROVED** [review]

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 12]
```diff
@@ -2,6 +2,49 @@
...
+ - Searching for a channel by URL now returns the channel.
+ - Users are now provided with feedback when creating a custom category name that exceeds the character limit.
+
+#### Administration
+ - Added support for compressed export files with attachments.
+ - Server crashes due to runtime panics are now captured as a log line.
+ - Optimized Direct Message creation by fetching all users involved in a single database call.
+ - During the user import process, a change in a user's ``NotifyProps`` will not send an email notification. This is done to make it consistent with other parts of the import process where a change in user's attributes would also not send any notifications.
+ - Implemented a job to delete unused export files.
+ - Improved the websocket implementation by using epoll to manually read from a websocket connection. As a result, the number of goroutines is expected to go down by half. This implementation is only available on Linux and FreeBSD based distributions. If you are using NGINX as a proxy to Mattermost, please ensure to have ``proxy_http_version 1.1;`` in the block that handles the websocket path.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:29]:
```suggestion
 - Improved the websocket implementation by using epoll to manually read from a websocket connection. As a result, the number of goroutines is expected to go down by half. This implementation is only available on Linux and FreeBSD-based distributions. If you are using NGINX as a proxy to Mattermost, please ensure to have ``proxy_http_version 1.1;`` in the block that handles the websocket path.
```

**Review by cwarnermm — COMMENTED** [review]
One minor proposed update added inline

**Comment by amyblais** [comment]
@esethna I'm removing custom status feature from the public changelog since we're planning to have it disabled in tomorrow's cloud release.

**Pull request merged**